### DISCUSSION TITLE
Create window from scratch

### DISCUSSION THREAD

**Discussion by Kemsekov** [AvaloniaUI/Avalonia#8874] [Q&A] [answered]
I am looking forward to using Avalonia to do some data visualization and I need to speed up launch time as much as possible as well as I don't need many features avalonia gives me. Can you give me a code that just creates a window without all other stuff binding into it and slowing down everything?

**Comment by timunie** [comment, marked as answer]
Try Avalonia 11.0 preview1. It's really fast. 

Also 
Use SimpleTheme. It's more light weight if that matters that much.

**Reply by Kemsekov** [reply]
Is there dotnet template for 11 version? If I update version to 11 it does not compile. I don't know how to create an application in 11 version. I working under linux so Visual studio templates are not working for me btw.

**Reply by timunie** [reply]
Not yet. You need to enable prerelease in your nuget feed and update to 11.0. after that find a package Avalonia.Themes.Simple and install this. Last but not least change FluentTheme to SimpleTheme in App.axaml.

**Reply by Kemsekov** [reply]
Ok, thanks, it worked out
